Bottom: 55d6233715ee1c6f028067720f776721bf59eb4f
Top:    2c1c23ba06a865593147be04400e4d1827f952f3
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-16 20:50:49 +0800

Refresh of vhost_net-rx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index d1490cc..b511cfd 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -992,10 +992,10 @@ static struct skb_array *get_tap_skb_array(int fd)
 	struct skb_array *array;
 	struct file *file = fget(fd);
 
-	if (!file) {
+	if (!file)
 		return NULL;
 	array = tap_get_skb_array(file);
-	fput(fd);
+	fput(file);
 	return array;
 }
